src/Nancy/Json/Simple/SimpleJson.cs

```diff
@@ -1237,6 +1238,25 @@ interface IJsonSerializerStrategy
         string MapDictionaryKeyToFieldName(string stringKey);
     }
 
+#if SIMPLE_JSON_INTERNAL

```

Comment:
The other classes in this file have the `[GeneratedCode("simple-json", "1.0.0")]` attribute applied to them while the new one didn't. I wasn't sure if this attribute was used for something internally or if it was just metadata for tooling.